server/src/utils.js

```diff
@@ -1,16 +1,46 @@
 const {DataSource} = require('apollo-datasource');
 
 class mainAPI extends DataSource {
-  constructor(db) {
+  constructor(store) {
     super();
-    this.db = db;
+    this.store = store;
   }
 
   initialize(config) {
     this.context = config.context;
   }
 
-  // TODO(arin-kwak): Implement mainAPI
+  async findOneUser(where) {
```

Comment:
findOne() is the db function name. We don't have to use this name for the API for downstream code - i.e. we can encapsulate how the db is structured. Let's call this findUser() ?

Reply:
Thank you! Fixed.